File: test/libsrcml/unit/xpath_test_data/is_private/is_private.java
private class Employee2 // 1
{
    private string name = "FirstName, LastName"; // 2

    private string GetName() //3
    {
        return name;
    }
    
    
    private class FooCls{ } //4
    
    class FooCls2 { } //5
}

class Employee2 // 6
{
}

interface IEmployee // 7
{
    string GetName();
}

File: test/libsrcml/unit/xpath_test_data/is_private/is_private.cs
class Employee2
{
    private string name = "FirstName, LastName";// 1

    private string GetName() //2
    {
        return name;
    }
    
    string GetName//3
    {
        private get; //4
        private set; //5
    }
    
    string GetName// 6
    {
        public get;
        set; // 7
    }
    
    private class FooCls { } // 8
    private struct FooStruct{ } // 9
    
    class FooCls2 { }  // 10
    struct FooStruct2 { } // 11
    
    event privateEvent PrivateEvent;//12
    private event privateEvent PrivateEvent;//13
}

interface IEmployee {
    string GetName();
    
    event publicEvent PublicEvent;
    private event privateEvent PrivateEvent;//14
    
    string GetName
    {
        get;
        set;
    }
}
